README.md

```diff
@@ -26,6 +33,75 @@
 		<img src="https://img.shields.io/github/forks/att-cloudnative-labs/kythera-k8s-deployment-crawler.svg?style=social">
 	</a>	
 </p>
+<p align="center">
+  <a href="https://goreportcard.com/badge/github.com/att-cloudnative-labs/kythera-k8s-deployment-crawler" alt="Go Report Card">
+    <img src="https://goreportcard.com/badge/github.com/att-cloudnative-labs/kythera-k8s-deployment-crawler">
+  </a>	
+</p>
+
+## Deployment
+Helm is required to install the Kyther Kubernetes Deployment Crawler. For information on installing Helm please refer to the [Helm quickstart guide](https://helm.sh/docs/using_helm/). After installing helm the following steps can be manually run
+
+1. Build docker image
+```bash
+$ docker build -t kubecrawler .
+```
+2. Run helm template to install Kubecrawler
+```bash
+$ helm template kubekleaner --set image=<KUBECRAWLER_IMAGE> | kubectl create -f -
+```
+
+Or the makefile can be used pulling values from ./helm/kubecrawler/values.yaml
+
+```bash
+$ make
```

Comment:
is the default target set? Or does it just know what to do if there's only one target?

Reply:
@timjonesdev Is this fine as is or does a target need to be set before merge?

Reply:
@timjonesdev 

> To run Make:

> $ make
> Make will look for a Makefile called Makefile and will build the default target, the first target in the Makefile.

Source: https://swcarpentry.github.io/make-novice/reference.html